bundle/config/resources.go

```diff
@@ -151,6 +151,12 @@ func (r *Resources) FindResourceByConfigKey(key string) (ConfigResource, error)
 		}
 	}
 
+	for k := range r.QualityMonitors {
+		if k == key {
+			found = append(found, r.QualityMonitors[k])
```

Comment:
nit: curious, why you're not using 2-element range here

Reply:
This is following the existing style of this file. It might need some refactoring once we add all resources to the group